### PR TITLE
Bug 982260 - Gaia-integration tests on TBPL fail with: TEST-UNEXPECTED-F...

### PR DESCRIPTION
**Pull request by adrm**
...AIL | Contacts > Details Merging 15 contacts

### CODE REVIEW

**Comment by try-server-hook** [comment]
[ADLR-es Adrián de la Rosa (ADLR-es)](https://github.com/ADLR-es) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=1711b2441781)

**Comment by try-server-hook** [comment]
[ADLR-es Adrián de la Rosa (ADLR-es)](https://github.com/ADLR-es) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=0edeed7d2d9e)

**Comment by try-server-hook** [comment]
[ADLR-es Adrián de la Rosa (ADLR-es)](https://github.com/ADLR-es) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=df893a74cd01)

**Comment by try-server-hook** [comment]
[ADLR-es Adrián de la Rosa (ADLR-es)](https://github.com/ADLR-es) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=84da57669346)

**Comment by try-server-hook** [comment]
[ADLR-es Adrián de la Rosa (ADLR-es)](https://github.com/ADLR-es) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=1a242d91acfc)

**Comment by try-server-hook** [comment]
[ADLR-es Adrián de la Rosa (ADLR-es)](https://github.com/ADLR-es) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=5c650b7ffeb5)

**Comment by try-server-hook** [comment]
[ADLR-es Adrián de la Rosa (ADLR-es)](https://github.com/ADLR-es) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=91096aed8737)

**Comment by try-server-hook** [comment]
[ADLR-es Adrián de la Rosa (ADLR-es)](https://github.com/ADLR-es) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=878afdd90d22)

**Comment by try-server-hook** [comment]
[ADLR-es Adrián de la Rosa (ADLR-es)](https://github.com/ADLR-es) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=4994fa00de8d)

**Comment by try-server-hook** [comment]
[ADLR-es Adrián de la Rosa (ADLR-es)](https://github.com/ADLR-es) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a8e9ca476bc9)

File: apps/communications/contacts/test/marionette/details_test.js [hunk 1]
```diff
@@ -1,142 +1,107 @@
...
+  test('Regular contact is displayed correctly', function() {
+    contactsData.createMozContact(testContact);
+    client.helper.waitForElement(selectors.listContactFirstText).click();
     subject.waitSlideLeft('details');
+    assertContactData(testContact);
 
-    subject.clickOn(client.helper.waitForElement(
-      selectors.detailsFindDuplicate));
+    var linkButton = client.findElement(selectors.detailsLinkButton);
+    linkButton.enabled();
```
Comment (jmcanterafonseca) [inline apps/communications/contacts/test/marionette/details_test.js]: we can simply check that the link button is present and we are done

**Comment by try-server-hook** [comment]
[ADLR-es Adrián de la Rosa (ADLR-es)](https://github.com/ADLR-es) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=296de024977c)

**Comment by try-server-hook** [comment]
[ADLR-es Adrián de la Rosa (ADLR-es)](https://github.com/ADLR-es) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=1eab0c82ac22)

**Comment by try-server-hook** [comment]
[ADLR-es Adrián de la Rosa (ADLR-es)](https://github.com/ADLR-es) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=32a9c18454e5)

**Comment by try-server-hook** [comment]
[ADLR-es Adrián de la Rosa (ADLR-es)](https://github.com/ADLR-es) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=2b94ccb9af64)

**Comment by try-server-hook** [comment]
[ADLR-es Adrián de la Rosa (ADLR-es)](https://github.com/ADLR-es) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=bf89be79559c)

**Pull request merged**